pyproject.toml

```diff
@@ -44,8 +44,9 @@ dependencies = [
     "click",
     "rich>=13.0.0",
     "rich-click",
+    "multipart@git+https://github.com/defnull/multipart.git",
```

Comment:
Pinned to `master` for now until a new release is out. See https://github.com/defnull/multipart/issues/68

Reply:
good switch, I didn't know that library and it looks very well thought, I'm always impressed by people talking bytes.